src/test/java/SpringJPA/AdminPageTest.java

```diff
@@ -1,35 +1,40 @@
-package SpringJPA;
-
-import static org.hamcrest.Matchers.containsString;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
-
-import org.junit.jupiter.api.Test;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.web.servlet.MockMvc;
-
-@SpringBootTest
-@AutoConfigureMockMvc
-public class TestingWebApplicationTest {
-
-    @Autowired
-    private MockMvc mockMvc;
-
-    @Test
-    public void shouldReturnDefaultMessage() throws Exception {
-        this.mockMvc.perform(get("/")).andDo(print()).andExpect(status().isOk())
-                .andExpect(content().string(containsString("Lorem ipsum dolor sit amet")));
-    }
-
-    @Test
-    public void shouldReturnLoginMessage() throws Exception {
-        this.mockMvc.perform(get("/login")).andDo(print()).andExpect(status().isOk())
-                .andExpect(content().string(containsString("Please sign in")));
-    }
-
-}
\ No newline at end of file
+package SpringJPA;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+
+@AutoConfigureMockMvc
+@SpringBootTest
+public class AdminPageTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    @WithMockUser(username="User1")
+    public void UserAPICallsAppears() throws Exception{
+        this.mockMvc.perform(get("/user/admin")).andDo(print()).andExpect(status().isOk())
+                .andExpect(content().string(containsString("0/1000")));
+    }

```

Comment:
This should be using An admin type user, not a normal one

Reply:
Sorry, I wasn't aware that there were different user types when I pulled from main. I figured I wouldn't need to change that until the role functionality was finalized. I'll make the change.